src/app/feature/campaign/campaign.service.ts

```diff
@@ -48,8 +47,13 @@ export class CampaignService {
     private localNotificationService: LocalNotificationService,
     private templateTranslateService: TemplateTranslateService,
     private appDataService: AppDataService,
+    private appConfigService: AppConfigService,
     private injector: Injector
-  ) {}
+  ) {
+    this.appConfigService.appConfig$.subscribe((appConfig: IAppConfig) => {
```

Comment:
Thanks for adding this to track the updates. Might be better to move into a `subscribeToAppConfigChanges` function or similar